lib/ui/confirmorder/confirm_order_page.dart

```diff
@@ -0,0 +1,182 @@
+                      Container(
+                          height: 35,
+                          child: ListTile(
+                              contentPadding: EdgeInsets.all(0),
+                              leading: new Checkbox(
+                                  value: reConfirmCheckBox,
+                                  onChanged: (bool value) {
+                                    setState(() {
+                                      reConfirmCheckBox = value;
+                                    });
+                                  }),
+                              title:
+                                  Text('Reconfirm order after store sets price',
+                                      style: GoogleFonts.openSans(
+                                        fontSize: 14,
+                                        color: Color(0XFF99879D),
+                                      )))),
+                      Container(
+                          height: 30,
+                          child: ListTile(
+                              contentPadding: EdgeInsets.all(0),
+                              leading: new Checkbox(
+                                  value: callMeChkBox,
+                                  onChanged: (bool value) {
+                                    setState(() {
+                                      callMeChkBox = value;
+                                    });
+                                  }),
+                              title: Text('Call me for any questions',
+                                  style: GoogleFonts.openSans(
+                                      fontSize: 14,
+                                      color: Color(0XFF99879D))))),
+                      SizedBox(height: 25),
+                      Center(
+                          child: ButtonTheme(
+                              minWidth: 50,
+                              height: 20,
+                              padding: EdgeInsets.fromLTRB(60, 10, 60, 10),
+                              child: RaisedButton(
+                                  onPressed: () {
+                                    saveOrder(context);
+                                  },
+                                  color: Color(0XFF9378FF),
+                                  child: Text('Confirm Order',
+                                      style: GoogleFonts.openSans(
+                                          color: Colors.white, fontSize: 24)),
+                                  shape: RoundedRectangleBorder(
+                                      borderRadius:
+                                          BorderRadius.circular(60)))))
+                    ]))));
+  }
+
+  @override
+  void onCreateOrderError(String errorText) {
+    scaffoldkey.currentState.showSnackBar(SnackBar(
+        content: Text('Network error while trying to create your order')));
+  }
+
+  @override
+  void onCreateOrderSuccess(Order order) {
+    print(order);
```

Comment:
Let's not have print statements!

Reply:
I have the print, since there is no order detail yet. Will remove when i have the order detail page